### DISCUSSION TITLE
Autocomplete 2.3.1

### DISCUSSION THREAD

**Discussion by ELF-VIGO** [heroui-inc/heroui#2753] [Feedback]
The behavior of the Autocomplete 2.3.1 has changed. Now, when you open the listbox, the input lost the focus.
Also, I have 3 Autocompletes in a modal and event behavior has change: now the change events doesn't work as before and it crash my component behavior.
Thank you very much for your work. I love these components. If I can help with this, please tell me to add more information.

**Comment by wingkwong** [comment]
Can you share some code or even a screen record for demonstration?

**Reply by ELF-VIGO** [reply]
You can see the first part at https://nextui.org/docs/components/autocomplete. If you click the control, first the list is open and you have to click twice to write into the Input.
I'll try to add some code about the second part. Sorry.

**Comment by ELF-VIGO** [comment]
I've updated and I think my issues are gone. Thanks.
